app/src/main/java/com/openlattice/chronicle/services/usage/UsageMonitoringWorker.kt

```diff
@@ -110,7 +111,15 @@ class UsageMonitoringWorker(context: Context, workerParameters: WorkerParameters
     }
 
     private fun getPropertyTypeIds(): Map<FullQualifiedName, UUID> {
-        return chronicleApi.getPropertyTypeIds(PROPERTY_TYPES) ?: ImmutableMap.of()
+        // try retrieving cached values first
```

Comment:
What's the reason for this change?

Reply:
If chronicle server is unreachable the device should still be able to collect data and store in the db for later upload.